resources/qml/ReplyPopup.qml

```diff
@@ -10,15 +10,16 @@ Rectangle {
     property var room: TimelineManager.timeline
 
     Layout.fillWidth: true
-    visible: room && room.reply
+    visible: room && (room.reply || room.edit)
     // Height of child, plus margins, plus border
-    implicitHeight: replyPreview.height + 10
+    implicitHeight: (room && room.reply ? replyPreview.height : closeEditButton.height) + 10
```

Comment:
I think room && is probably superfluous here?  Since the popup is only visible if 'room' is defined, I'm thinking the height doesn't really matter much.

Reply:
If you don't do the room &&, you get null warnings. It's stupid, but sadly necessary. It's just in the condition to prevent the null warning, apart from this the conditional sets this to reply height or edit height, depending o. if this has a reply or not.